package.json

```
@@ -53,7 +53,7 @@
     "jest": "^27.5.1",
     "lint-staged": ">=10",
     "pa11y-ci": "^3.0.1",
-    "prettier": "2.6.1",
+    "prettier": "^2.6.1",
```

I think this should stay pinned to a specific version (this is how prettier suggests you install it)

```suggestion
    "prettier": "2.6.1",
```